.gitignore

```diff
@@ -0,0 +1,11 @@
+#
```

Comment:
These entries should probably live in your global `gitignore` file.

https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

Reply:
@x80486 I think the suggestion from @joschi is a good one. If you would be so kind as to remove the .gitignore, I'll merge your PR ASAP. Thank you!